component/service/src/test/java/org/exoplatform/social/service/malwareDetection/MalwareDetectionServiceTest.java

```diff
@@ -0,0 +1,27 @@
+package org.exoplatform.social.service.malwareDetection;
+
+import org.mockito.Mockito;
+
+import org.exoplatform.social.service.malwareDetection.connector.MalwareDetectionServiceConnector;
+import org.exoplatform.social.service.test.AbstractResourceTest;
+
+public class MalwareDetectionServiceTest extends AbstractResourceTest {
+  
+  private MalwareDetectionService malwareDetectionService;
+
+  public void setUp() throws Exception {
+    super.setUp();
+    malwareDetectionService = getContainer().getComponentInstanceOfType(MalwareDetectionService.class);
+  }
+
+  public void testProcessInfectedItem() {
+    String infectedItemPath = "/gatein/data/files/d/e/4/c/e/2/d/7/dedeerere673r777";
```

Comment:
Can you make a test with more than one infected item

Reply:
Right now the test done is for the MalwareDetectionService which process only one infected item. We can test MalwareDetectionServiceRestService also in order to test the case of a list of infected items